### ISSUE TITLE
Assess compliance with HO Accessibility Standard

### ISSUE THREAD

**Issue by edhamiltonHO** [HO-CTO/engineering-guidance-and-standards#60]
The [Home Office Accessibility Standard](https://design.homeoffice.gov.uk/accessibility/standard)

This site should be reviewed and issues raised to resolve where the standard is not met

- [x] #132  
- [x] #137  
- [x] #138 
- [x] #139

**Comment by edhamiltonHO** [comment]
- [ ] Implement automated aXe tests
- [ ] create checklist to include in PR templates
- [ ] create accessibility statement
- [ ] run through manual test checklist

**Comment by edhamiltonHO** [comment]
All sub items complete so closing this issue
